### ISSUE TITLE
School Performance Data

### ISSUE THREAD

**Issue by keeganmcbride** [okestonia/Estonian-Open-Data-Issue-Tracker#35]
A dataset which contained information regarding schools' performance would be beneficial to have.

**Comment by tormi** [comment]
What do you mean by `performance`? Building's performance (square meters per pupil etc) or some qualitative performance?

**Comment by McSim** [comment]
Daily marks and absents by the schools, classes, teachers and subjects. Without names. With results by terms.
I think that even such limited set of features would be an excellent source for many different investigates.

**Comment by keeganmcbride** [comment]
As an example, in the state of Georgia they have this: https://schoolgrades.georgia.gov/

"This website provides school reports for all public elementary, middle, and high schools in Georgia. These reports include A-F letter grades based on school performance and other useful information about the school, such as performance on statewide assessments, the make-up of the school’s student body, the graduation rate, and additional academic information. The Governor’s Office of Student Achievement (GOSA) creates these reports to provide parents with clear and concise information on school performance for all stakeholders."

**Comment by tormi** [comment]
I don't think we have such grades. Best I could find is this: http://www.delfi.ee/misc/koolid/

**Comment by McSim** [comment]
We have the system 'ekool'. It would be nice to make negotiations with the company Koolitööde AS.

**Comment by andreskytt** [comment]
ekool is but one of the providers. Would the results of state exams be valuable? 

For example:
 * School
 * Gender (?)
 * Exam type
 * Grade
 * Date/time

**Comment by Hillehinsberg** [comment]
Ministry has this site for education statistics http://qlikview-pub.hm.ee/QvAJAXZfc/opendoc_hm.htm?document=htm_avalik.qvw&host=QVS%40qlikview-pub&anonymous=true

**Comment by ruizcalleja** [comment]
Maybe this can serve as an example? https://schoolgrades.georgia.gov

**Comment by andreskytt** [comment]
@keeganmcbride, would the columns described above be useful? This data I know to be there and am suspicious much more is almost impossible to get

**Comment by joellimberg** [comment]
Some generic background/thoughts: comparing marks given by teachers **across schools** is not really a reliable indicator — schools have different marking schemes and _"strictness"_.

Standardized testing results could be compared, though (the state has this data).

**Comment by keeganmcbride** [comment]
@joellimberg  Yes, at this point it seems that the state standardized exam performance data would be what could work for us, you make a good point about the reliability issue.

**Comment by joellimberg** [comment]
@keeganmcbride I couldn't find this in English, but _Innove_ also publishes quite extensive state exam statistics, by subject and school: http://www.innove.ee/et/riigieksamid/riigieksamite-statistika/riigieksamite-statistika-2016

For example, the Estonian language exam: http://www.innove.ee/UserFiles/Riigieksamid/2016/Statistika%20veebilehele/eesti_keele_riigieksam_2016_veebi(2).html

**Comment by taivop** [comment]
@andreskytt I'm interested in the columns you described (state exam results ).

Whom should I ask? My default would be to send an official information request, but I'm slightly afraid they'd refer to vague privacy concerns and not release person-level data.

**Comment by KarinLeivategija** [comment]
@andreskytt the data collected by eKool would be interesting. Is there a chance the company would share it? Not just exam results, but data that reflects everyday life in schools - grades, absences etc. - with privacy concerns taken into account.

**Comment by infokujur** [comment]
Is there any info what HMT has but is not publishing here regarding to this topic? https://opendata.riik.ee/andmehulgad/?organization=haridus-ja-teadusministeerium

No progress on e-kool AFAIK

**Comment by andreskytt** [comment]
> @andreskytt the data collected by eKool would be interesting. Is there a chance the company would share it? Not just exam results, but data that reflects everyday life in schools - grades, absences etc. - with privacy concerns taken into account.

Very unlikely, they perceive this data to be their competitive advantage.

**Comment by joellimberg** [comment]
Background: we run a competing e-school platform Stuudium (https://stuudium.com/)

Sorry for maybe being rude, but I don’t understand how one would consider (for example) data about feedback given to students to be “open data” in any way.

We at Stuudium would also not share such data, but not because it’s a “competitive advantage” (how? In theory, what would a service provider lose by giving out / selling — as a grossly simplified example — a bunch of student report cards without names attached?)

Grading in schools is not really standardized, and schools are moving away from measuring performance with numbers — grades are being replaced by written personal feedback.

Also, the more personalized education and feedback gets, the harder it is to properly anonymize. (Simply removing names is certainly not enough)

In my opinion it would be a large step back to start somehow “measuring performance” based on _big data_.

The only exception to the above would be aggregated absence data, which might give some insight about illness outbreaks/etc. An alternative route to this data might be related to the school meal funding (this is handled by a combination of state and local gov.)

**Comment by peepkungas** [comment]
What about data, which is used here: https://digipeegel.ee - would this make sense in the context of school performance?

**Comment by andreskytt** [comment]
> What about data, which is used here: https://digipeegel.ee - would this make sense in the context of school performance?

Well, this is mostly about digital maturity. Even so, they should make the data available. @infokujur, this is a good example of science data being closed by default.
